packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -535,90 +542,132 @@ export class JobService {
     }
   }
 
-  public async cancelCronJob() {
+  public async refundCronJob() {
```

Comment:
why we split cancel and refund logic? We need to refund right after cancel when the funds have been received in the job launcher account

Reply:
https://github.com/humanprotocol/human-protocol/pull/869#issuecomment-1713799233

Reply:
If we want to return the amount that will be released from the escrow address, which we can receive in the escrow logs, then the logs will not yet be available.